testutil/compose/compose/main.go

```diff
@@ -281,6 +281,7 @@ func newNewCmd() *cobra.Command {
 	extBootnode := cmd.Flags().String("external-bootnode", "", "Optional external bootnode HTTP url.")
 	splitKeys := cmd.Flags().String("split-keys-dir", conf.SplitKeysDir, "Directory containing keys to split for keygen==create, or empty not to split.")
 	featureSet := cmd.Flags().String("feature-set", conf.FeatureSet, "Minimum feature set to enable: alpha, beta, stable")
+	numVals := cmd.Flags().Int("num-validators", conf.NumValidators, "Number of distributed validators.")
```

Comment:
ensure there is a default defined in `compose.NewDefaultConfig()`

Reply:
yup there is already a default defined as `defaultNumVals`